tests/unit/test_service_steps.py

```diff
@@ -874,8 +968,8 @@ def test_eks_delete_fargate_profile_step_creation():
 
 
 @patch.object(boto3.session.Session, 'region_name', 'us-east-1')
-def test_eks_delete_fargate_profile_step_creation_sync():
-    step = EksDeleteFargateProfileStep("Delete Fargate profile sync", parameters={
+def test_eks_delete_fargate_profile_step_creation_default():
+    step = EksDeleteFargateProfileStep("Delete Fargate profile Default", parameters={
```

Comment:
nit: multiple locations - doesn't need capitalization :)
```suggestion
    step = EksDeleteFargateProfileStep("Delete Fargate profile - default", parameters={
```